packages/suite-data/src/translations/list-unused.ts

```diff
@@ -3,7 +3,6 @@ import path from 'path';
 import fs from 'fs';
 
 // See comment in list-duplicates.ts
-// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
Oh no.. I saw the comment in `list-duplicates.ts` :see_no_evil: Quite a hack, but that's out of scope of this PR.
More importantly - isn't it an issue that eslint no longer gives you error? :thinking: